python/mlc_chat/model/utils/kv_cache.py

```diff
@@ -0,0 +1,118 @@
+"""Attention KV cache modeling."""
+from tvm import relax as rx
```

Comment:
move to `python/mlc_chat/nn/kv_cache.py`, it's just like PyTorch has `Linear` under `nn`

Reply:
Moved.